.travis.yml

```diff
@@ -1,10 +1,15 @@
 language: php
 sudo: false
 
+## Cache composer bits
+cache:
+  directories:
+    - $HOME/.composer/cache
 php:
-  - 5.5
   - 5.6
   - 7.0
 
 before_script:
-  - cd tests/
+  - composer install --prefer-dist
```

Comment:
I flip flopped over this, but the logic I used on myself was `use a dependency manager then you know that the tests are running in the same environment`.